app/services/create_batch.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
```

Comment:
This could be more expressive. I have to read the AR code to try and work what type of `AppointmentSummary` these are.

We could choose a more descriptive variable name and/or we could scope this maybe?

```rb
class AppointmentSummary
  scope :unprocessed, -> do
    where('appointment_summaries_batches.appointment_summary_id': nil)
      .includes(:appointment_summaries_batches)
  end
end

unprocessed_appointment_summaries = AppointmentSummary.unprocessed
```